packages/kbn-profiling-utils/common/profiling.ts

```diff
@@ -46,15 +47,35 @@ const frameTypeDescriptions = {
   [FrameType.Perl]: 'Perl',
   [FrameType.JavaScript]: 'JavaScript',
   [FrameType.PHPJIT]: 'PHP JIT',
+  [FrameType.Error]: 'Error',
 };
 
+/**
+ * normalize the given frame type
+ * @param ft FrameType
+ * @returns FrameType
+ */
+export function normalizeFrameType(ft: FrameType): FrameType {
+  // Normalize any frame type with error bit into our uniform error variant.
+  if (((ft as number) & 0x80) != 0) {
```

Comment:
And you should aways use `!==` to compare.
```suggestion
export function normalizeFrameType(ft: FrameType): FrameType {
  // Normalize any frame type with error bit into our uniform error variant.
  if (((ft as number) & 0x80) !== 0) {
```